.meshrc.yaml

```diff
@@ -244,9 +250,9 @@ additionalTypeDefs: |
   }
 additionalResolvers:
   - ./resolvers.ts
-plugins:
-  - csrfPrevention:
-      requestHeaders: ['x-graphql-yoga-csrf']
+# plugins:
+#   - csrfPrevention:
+#       requestHeaders: ['x-graphql-yoga-csrf']
```

Comment:
we need to leave this enabled.  If you merge in `main`, this header is no longer required for queries made in the graphQL playground though.